website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -73,12 +73,50 @@ $ operator-sdk create api \
 **Note** Ensure that you use the same values for the flags to recreate the same Helm Chart and API's. If you have
 more than one chart or API's you can add them via `operator-sdk create api` command. For further information check the [quick-start][quickstart]. 
  
-### Replacing the content
+### Migrating your CR's
 
-- Update the CR manifests in `config/samples` with the values of the CR's in your old project which are in `deploy/crds/`
-- Check if you have customizations options in the `watch.yaml` file of your previous project and then, update the new `watch.yaml` file with the same ones
-- Ensure that all roles configured in the `/deploy/roles.yaml` will be applied in the new project in the file `config/rbac/role.yaml`
-- If you have customizations in your `helm-charts` then, apply them in the new `helm-charts`. Note that this directory was not changed at all.
+Update the CR manifests in `config/samples` with the values of the CR's in your old project which are in `deploy/crds/`.
+
+### Migrating your Watches
+Check if you have customizations options in the `watch.yaml` file of your previous project and then, update the new `watch.yaml` file with the same ones. In our example, it will look like:
+
+```yaml
+# Use the 'create api' subcommand to add watches to this file.
+- group: example.com
+  version: v1alpha1
+  kind: Nginx
+  chart: helm-charts/nginx
+# +kubebuilder:scaffold:watch
+```
+
+**NOTE**: Do not remove the [Maker][marker] `+kubebuilder:scaffold:watch`. It allows the tool to update the watches file when new API's are created. 
+
+### Checking the Permissions (RBAC)
+
+SDK generated the roles for you in the `config/rbac/role.yaml`. However, you might need to  
+update this file with the permissions which are in your previous project in `/deploy/roles.yaml`.
+
+And then, beware that:
+
+- By default, the projects are `cluster-scoped`. Check that your role is of the `ClusterRole` Kind. For further information check the [operator-scope][operator-scope] doc.  
+
+- The `apiGroups` for `monitoring.coreos.com` is no longer required. If you would like to export the Prometheus metrics check the next section `Exporting metrics`.
```

Comment:
All perfect :-) But I still think that we need to add a note to highlight that `The `apiGroups` for `monitoring.coreos.com` is no longer required` otherwise, could not users face permissions issues and decide to port it thinking that it is the root cause?

Reply:
I just made a change here. Instead o describe the actions/resource I added the piece from the yaml file. I think it will make the user's life easier.